### DISCUSSION TITLE
How often should stable releases come out

### DISCUSSION THREAD

**Discussion by jdev082** [CatalystDevOrg/Catalyst#5] [General]
Weekly
Bi-Weekly (Every 2 Weeks)
Tri-Weekly (Every 3 Weeks)
Quad-Weekly (Every 4 weeks)
Every 6 weeks
Monthly
Bi-Monthly
Yearly
Bi-Yearly

**Comment by hello-smile6** [comment]
I'd vote every 4 weeks, as that seems to work for Chromium, and that gives plenty of time to resolve feuds between warring contributors without disrupting releases.
(If you agree, just upvote this comment. Don't make a new thread.)

**Comment by hello-smile6** [comment]
@JaydenDev Could you make this a Q&A discussion? (And hide this comment, as it's off-topic)

**Comment by VeryUsual** [comment]
Bi-weekly

**Comment by jdev082** [comment]
I've decided on quad weekly
Daily bug fixes

**Reply by hello-smile6** [reply]
Nice!

**Comment by icarusinthewings** [comment]
Monthly, with versioning numbers like 21.12 (December 2021), 22.01 (January 2022), 22.01.1 (Patch 1 of Jan. 2022), etc.

**Reply by jdev082** [reply]
the 21.12 stuff goes to daily and dev builds. 1.0, 2.0 versioning goes to pre releases, betas, and stable releases.
However, I could make 21.12 stuff be the releases and everything else have that third number

**Reply by jdev082** [reply]
21.12.100-beta
21.12-prerel-3
21.12.124-dev
21.12-stable

**Reply by imdevarsh** [reply]
For the tag names @JaydenDev please don't add any characters in the tag name other than the `v` at the start, otherwise it will make the auto updator mad.

**Comment by imdevarsh** [comment]
Maybe make a team of "testers" who might not want to code but can test the app and find bugs?

**Reply by jdev082** [reply]
@webdev03 I already test every Catalyst release by using it daily lmao

**Reply by imdevarsh** [reply]
lol
